pkg/controllers/dynakube/controller_test.go

```diff
@@ -356,13 +360,14 @@ func TestReconcileComponents(t *testing.T) {
 			versionReconcilerBuilder:        createVersionReconcilerBuilder(mockVersionReconciler),
 			connectionInfoReconcilerBuilder: createConnectionInfoReconcilerBuilder(mockConnectionInfoReconciler),
 			activeGateReconcilerBuilder:     createActivegateReconcilerBuilder(mockActiveGateReconciler),
+			injectionReconcilerBuilder:      createInjectionReconcilerBuilder(mockInjectionReconciler),
 		}
 		mockedDtc := mockedclient.NewClient(t)
 		err := controller.reconcileComponents(ctx, mockedDtc, nil, dynakube)
 
 		require.Error(t, err)
 		// goerrors.Join concats errors with \n
-		assert.Len(t, strings.Split(err.Error(), "\n"), 4) // ActiveGate, OneAgentInjection, EnrichmentInjection, OneAgent
+		assert.Len(t, strings.Split(err.Error(), "\n"), 3) // ActiveGate, OneAgentInjection, EnrichmentInjection, OneAgent
```

Comment:
could you please update the comment? This assert is confusing now

Reply:
You are totally right! [Fixed here](https://github.com/Dynatrace/dynatrace-operator/pull/2736/commits/2541a78f1e4b7aa0fa0625e5190ba0a5c5824aba)